### PR TITLE
Added readme.md hello world

### PR DESCRIPTION
**Pull request by proznerka**
- [DEMO LINK](https://proznerka.github.io/layout_hello-world/)
- [TEST REPORT LINK](https://proznerka.github.io/layout_hello-world/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
```
Comment (VinnieJ-2k20) [inline src/index.html:11]: Content of a tag should also be intended.
```suggestion
  <body>
    Hello, world!
  </body>
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Almost done, just one small thing to fix.

File: src/index.html (hunk 1 above)
Comment (proznerka) [reply src/index.html:11]: image

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**